pkg/cmd/server/start/start_scheduler.go

```diff
@@ -0,0 +1,76 @@
+package start
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/origin/pkg/cmd/server/origin"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+)
+
+var schedulerLong = templates.LongDesc(`
+	Start the master scheduler
+
+	This command starts the scheduler for the master.  Running
+
+	    %[1]s start master %[2]s
+
+	will start the the scheduler. The scheduler will run in the foreground until you terminate the process.`)
+
+// NewCommandStartMasterScheduler starts only the scheduler
+func NewCommandStartMasterScheduler(name, basename string, out, errout io.Writer) (*cobra.Command, *MasterOptions) {
```

Comment:
We are trying to remove `openshift start *`.  I'd rather not see this added here.  What about `hyperkube kube-scheduler` with the right args?